### DISCUSSION TITLE
Disable sorting by access level

### DISCUSSION THREAD

**Discussion by SergeySmirnov-Akvelon** [codecadwallader/codemaid#830] [Ideas]
We are working in Microsoft repos that use [internal guidelines](https://docs.microsoft.com/en-us/archive/blogs/brada/internal-coding-guidelines#28-file-organization) that don't take access level into account when sorting type members (i.e. properties, methods, fields, etc.). Is there some way to exclude the access level from the sorting mechanism? Currently, I see that it is possible to select the direction of sorting by access level, but not exclude it.

**Comment by codecadwallader** [comment]
Thanks for asking the question.  We do not currently have an option to exclude it but that was requested once before in #569 .
